sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.1.0-preview1 (2020-03-10)
+
+- Extended DefaultAzureCredential with an experimental credential that uses the login credential from Azure CLI
+
 ## 1.0.3 (Unreleased)
```

Comment:
should we remove `## 1.0.3 ...` header? or we still need to release 1.0.x somehow?

Reply:
I think it's probably fine